.travis.yml

```diff
@@ -1,8 +1,11 @@
 
 language: node_js
+os:
+  - linux
+  - osx
 node_js:
-  - "5"
+  - "8"
```

Comment:
Huh, interesting! Certainly seems to work: https://travis-ci.org/resin-io-modules/resin-discoverable-services/jobs/253118857 :smile:

Looks like their docs are out of date: https://github.com/travis-ci/docs-travis-ci-com/pull/1278